libcxx/cmake/config-ix.cmake

```diff
@@ -27,6 +27,9 @@ if (NOT LIBCXX_USE_COMPILER_RT)
   endif()
 endif()
 
+check_cxx_compiler_flag(-nostdlibinc CXX_SUPPORTS_NOSTDLIBINC_FLAG)
+check_cxx_compiler_flag(-nolibc CXX_SUPPORTS_NOLIBC_FLAG)
```

Comment:
Those don't seem needed here, at least not in this iteration of the patch.

Reply:
They're used now that I moved `-nostdlibinc` and `-nostdlib` to the interface target.